### PR TITLE
Fix typos across whole repo using typos-cli and ruplacer

### PR DESCRIPTION
**Pull request by cmdcolin**
Hi there
I am not sure if this is a proper way to contribute this PR, but i just did some random work to fix typos. it is an idle hobby. if you would prefer this to split into many individual PR, or individual commits with one fix each, let me know. additionally, if upstream ontologies need the (some) fixes rather than here, let me know as well

I used typos-cli (https://crates.io/crates/typos-cli) and ruplacer (https://github.com/your-tools/ruplacer)

typos-cli alerted spelling errors, and then i could use ruplacer individually on certain terms that seemed correct to replace

therefore, all fixes to this repo constituted of the following ruplacer commands

```
ruplacer processs process --go
ruplacer "syndrom " "syndrome " --go
ruplacer modifed modified --go
ruplacer Brazillian Brazilian --go
ruplacer producting  producing --go
ruplacer exemple example --go
ruplacer insuffiency  insufficiency --go
ruplacer theraputic therapeutic --go
ruplacer tendancy tendency --gop
ruplacer tendancy tendency --go
ruplacer appearence appearance --go
ruplacer occuring occurring --go
ruplacer stucture structure --go
ruplacer funtion function --go
ruplacer inclusing  including --go
ruplacer Vomitting Vomiting --go
ruplacer Unusal Unusual --go
ruplacer Enlargment Enlargement --go
ruplacer theraputic  therapeutic --go
ruplacer betweem between --go
ruplacer perfom perform --go
ruplacer includ include --go
ruplacer includee include --go
ruplacer refering referring --go
ruplacer segements segments --go
ruplacer severly severely --go
ruplacer transmissable transmissible --go
ruplacer possibilites possibilities --go
ruplacer beahvior behavior --go
ruplacer actuall actual --go
ruplacer "actuall " "actual " --go
ruplacer organsim  organism --go
ruplacer resistence resistance --go
ruplacer membrance  membrane --go
ruplacer developement  development --go
ruplacer descreases decreases --go
ruplacer includeing including --go
ruplacer intoxination intoxication --go
ruplacer anomolous anomalous --go
ruplacer Intellecutal Intellectual --go
ruplacer acquistion acquisition --go
ruplacer enviroment environment --go
ruplacer wih with  --go
ruplacer "fron " "from " --go
ruplacer commited committed --go
ruplacer analagous  analogous --go
ruplacer preferrably preferably --go
ruplacer infered inferred --go
ruplacer distingushing distinguishing --go
ruplacer Adresses Addresses --go
ruplacer exlcude exclude --go
ruplacer columnn column --go
ruplacer Relace Replace --go
ruplacer properies properties --go
ruplacer obsolesence obsolescence --go
ruplacer wether whether --go
ruplacer dervied derived --go
ruplacer explaning explaining --go
ruplacer postive positive --go
ruplacer Commmit Commit --go
ruplacer defintion definition --go
ruplacer precurosr precursor --go
ruplacer maintainance maintenance --go
ruplacer apetite appetite --go
ruplacer corrresponding corresponding --go
ruplacer deprected deprecated --go
ruplacer querry query --go
ruplacer primairly primarily --go
ruplacer peple people --go
ruplacer laguage language --go
ruplacer abilty ability --go
ruplacer recgonize recognize --go
ruplacer "hte " "the "  --go
ruplacer hypertropy hypertrophy --go
ruplacer occurence occurrence --go
ruplacer defintion definition --go
ruplacer opthalmic ophthalmic --go
ruplacer tranform transform --go
ruplacer indispensible indispensable --go
ruplacer instrinsic intrinsic  --go
ruplacer auxillery auxiliary --go
ruplacer mucuous mucous --go
ruplacer divisons divisions --go
ruplacer colummn column --go
ruplacer kown known --go
ruplacer thryoid thyroid --go
ruplacer incudes  includes --go
ruplacer wtih with --go
ruplacer disproportionatley disproportionately --go
ruplacer systen system --go
ruplacer strcutural structural --go
ruplacer organim organism --go
ruplacer intoto into --go
ruplacer accoustic acoustic --go
ruplacer  Additonal   Additional  --go
ruplacer " oder " " order " --go
ruplacer Additionaly Additionally --go
ruplacer characterisitcs characteristics --go
ruplacer persistance persistence --go
ruplacer "hight metastatic" "high metastatic" --go
ruplacer hight "height" --go
ruplacer sublcass subclass --go
ruplacer protruberances protuberances --go
ruplacer intellecutal intellectual --go
ruplacer ocurring occurring --go
ruplacer superfical superficial --go
ruplacer consituents constituents --go
ruplacer assymetric asymmetric --go
ruplacer occurances occurrences --go
ruplacer familly family --go
ruplacer abscence absence --go
ruplacer damange damage --go
ruplacer inlcude include --go
ruplacer consitent consistent --go
ruplacer ocasionally occasionally --go
ruplacer impossibilty impossibility --go
ruplacer boundries boundaries --go
ruplacer differnt different --go
ruplacer consituents constituents --go
ruplacer persistant persistent --go
ruplacer aquired acquired --go
ruplacer invovle involve --go
ruplacer varation variation --go
ruplacer continously continuously --go
ruplacer trantion transition --go
ruplacer transtion transition --go
ruplacer transmitts transmits --go
ruplacer occured occurred --go
ruplacer appereance appearance --go
ruplacer speicific specific --go
ruplacer primarly primarily --go
ruplacer entitiy entity --go
ruplacer identicial identical --go
ruplacer insuficiency insufficiency --go
ruplacer predomiantly predominately --go
ruplacer triggerd triggered --go
ruplacer appereance appearance --go
ruplacer discernable discernible --go
ruplacer tihs this --go
ruplacer distint  distinct --go
ruplacer ocurr occur --go
ruplacer "appar- ent" "apparent" --go
ruplacer "Scondary" "Secondary" --go
ruplacer " wth " " with " --go
ruplacer Agressive Aggressive --go
ruplacer Intelectual Intellectual --go
ruplacer inflamation inflammation --go
ruplacer predominently predominantly --go
ruplacer seprate separate --go
ruplacer androgeny androgyny --go
ruplacer distrubance disturbance --go
ruplacer intial initial --go
ruplacer unpredicatable unpredictable --go
ruplacer inculding including --go
ruplacer condidtion condition --go
ruplacer "aparent" "a parent"  --go
ruplacer includea include --go
ruplacer disablity disability --go
ruplacer saggital sagittal --go
ruplacer sytem system --go
ruplacer opiods opioids --go
ruplacer condidtion condition --go
ruplacer occurence occurrence --go
ruplacer havea "have a" --go
ruplacer treatement treatment --go
ruplacer adbominal  abdominal --go
ruplacer snydrome syndrome --go
ruplacer sheat sheath --go
ruplacer sheathh sheath --go # this fixed above command double adding an h for already good spellings
ruplacer occurent occurrent --go
```

### CODE REVIEW

File: docs/artifacts/all-mondo-disease-terms.tsv
Comment (matentzn) [inline docs/artifacts/all-mondo-disease-terms.tsv:1]: exclude from PR, autogenerated

File: Changes.md
Comment (matentzn) [inline Changes.md:1]: exclude from PR, autogenerated.

File: docs/editors-guide/patterns/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@
...
+| [disease realized in response to environmental exposure](realized_in_response_to_environmental_exposure/) | This pattern is used for a disease, where the cause of the disease is an exposure to an environmental stimulus (using ECTO exposure terms) or a medical action, such as a therapeutic treatment (using MAxO terms). Note that this pattern does not include infectious disease or classes that would include an organism, virus or viroid. Rather it includes exposures to chemicals (includeng drugs), or mixtures.<br/>Examples: [chemically-induced disorder](http://purl.obolibrary.org/obo/MONDO_0029001), [alcohol amnestic disorder](http://purl.obolibrary.org/obo/MONDO_0021702), [alcoholic polyneuropathy](http://purl.obolibrary.org/obo/MONDO_0006645), [chemotherapy-induced alopecia](http://purl.obolibrary.org/obo/MONDO_0005483) (26 total) |
```
Comment (matentzn) [inline docs/editors-guide/patterns/index.md:34]: wrong fix

File: docs/editors-guide/patterns/realized_in_response_to_environmental_exposure.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 [http://purl.obolibrary.org/obo/mondo/patterns/realized_in_response_to_evironmental_exposure.yaml](http://purl.obolibrary.org/obo/mondo/patterns/realized_in_response_to_evironmental_exposure.yaml)
 ## Description 
 
-This pattern is used for a disease, where the cause of the disease is an exposure to an environmental stimulus (using ECTO exposure terms) or a medical action, such as a therapeutic treatment (using MAxO terms). Note that this pattern does not include infectious disease or classes that would include an organism, virus or viroid. Rather it includes exposures to chemicals (includng drugs), or mixtures.
+This pattern is used for a disease, where the cause of the disease is an exposure to an environmental stimulus (using ECTO exposure terms) or a medical action, such as a therapeutic treatment (using MAxO terms). Note that this pattern does not include infectious disease or classes that would include an organism, virus or viroid. Rather it includes exposures to chemicals (includeng drugs), or mixtures.
```
Comment (matentzn) [inline docs/editors-guide/patterns/realized_in_response_to_environmental_exposure.md:6]: wrong fix

File: imports/chebi_import.obo
Comment (matentzn) [inline imports/chebi_import.obo:1]: Exclude from PR, autogenerated.

File: imports/chebi_import.owl
Comment (matentzn) [inline imports/chebi_import.owl:1]: Exclude from PR, autogenerated.

File: imports/cl_import.obo
Comment (matentzn) [inline imports/cl_import.obo:1]: Exclude from PR, autogenerated.

File: imports/cl_import.owl
Comment (matentzn) [inline imports/cl_import.owl:1]: Exclude from PR, autogenerated.

File: imports/ecto_import.obo
Comment (matentzn) [inline imports/ecto_import.obo:1]: Exclude from PR, autogenerated.

File: imports/ecto_import.owl
Comment (matentzn) [inline imports/ecto_import.owl:1]: Exclude from PR, autogenerated.

File: reports/basic-report.tsv
Comment (matentzn) [inline reports/basic-report.tsv:1]: Exclude from PR, autogenerated.

File: reports/class-stats.tsv
Comment (matentzn) [inline reports/class-stats.tsv:1]: Exclude from PR, autogenerated.

File: scratch/VICC.json
Comment (matentzn) [inline scratch/VICC.json:1]: Exclude from PR, historical record.

File: scratch/vicc.obo
Comment (matentzn) [inline scratch/vicc.obo:1]: Exclude from PR, historical record.

**Review by matentzn — CHANGES_REQUESTED** [review]
Thank you for the drive by! :) Much appreciated. 

Many files you fixed typos in are auto generated and should not be fixed this way, see below. The most effective files to fix are

- `docs/*`
- src/ontology/mondo-edit.obo
- All README.md files

Instead of fixing absolutely everything, could you recreate the PR focusing on these?

Exclude all files in the following dirs, as they are all auto generated:

- src/ontology/imports
- imports
-  src/ontology/mappings
- src/ontology/reports
- subsets
- src/ontology/subsets
-

**Comment by cmdcolin** [comment]
Thank you for the detailed review @matentzn ! I just went ahead and made the fixes in a new PR here, pulling only the docs directory and mondo-edit.obo https://github.com/monarch-initiative/mondo/pull/7685

**Comment by cmdcolin** [comment]
will close this one for now

**Pull request closed without merging**